src/cm_adapter.js

```diff
@@ -1257,7 +1257,7 @@ class CMAdapter {
   }
 
   smartIndent() {
-    editor.getAction("editor.action.formatSelection").run();
+    this.editor.getAction("editor.action.formatSelection").run();
```

Comment:
```suggestion
    this.editor.getAction("editor.action.reindentselectedlines").run();
```

Not all languages have formatters defined and `=` doesn't mean format, it means indent.